app/models/concerns/dob.rb

```diff
@@ -0,0 +1,22 @@
+module Dob
```

Comment:
I would prefer **either** spell out the words:
```ruby
module DateOfBirth; end
```
**or** a capitalised version (you would need to add it the acronyms in `config/initializers/inflections.rb` file so that rails does not complain when doing the autoloding
```ruby
module DOB; end
```